### DISCUSSION TITLE
no bootable device

### DISCUSSION THREAD

**Discussion by imhessam** [ipxe/ipxe#1032] [Q&A]
hi
im trying to load a windows on the eve-ng.
I added the windows to eve and when start the windows see this message on the vnc:

net0: 50:00:00:03:00:00 using 82540em on 0000:00:03.0 (open)
  [Link:up, TX:0 TXE:0 RX:0 RXE:0]
Configuring (net0 50:00:00:03:00:00)...... ok
net0: 192.168.133.139/255.255.255.0 gw 192.168.133.2
Nothing to boot: No such file or directory (http://ipxe.org/2d03e13b)
No more network devices

No bootable device. 
![Screenshot (25)](https://github.com/ipxe/ipxe/assets/99562445/53f15c1f-56c1-4b95-95cd-626851b99bdc)

and this is my windows configure on eve:
![Screenshot (26)](https://github.com/ipxe/ipxe/assets/99562445/994c4de7-af7b-4958-9ec1-971e703c3f2e)

I tried multipie versions of windows. i use eve-ng ce and  i did it based on the eve-ng ce book.

can anyone help?!

**Comment by NiKiZe** [comment]
Probably not a cause for your direct issue, but you are using a quite old version of iPXE.

Have you tried reading the page shown in the message you get? http://ipxe.org/2d03e13b
That page describes exactly why you get the message:

> This error generally indicates that the DHCP server has not provided either a filename or a root-path for iPXE to boot from.

You need to make your DHCP send the right options. It's not clear which kind of network you have setup for your guest.
If you are using Qemu to provide the network then these options can be added there. But you might have to check the eve-ng documentation for how to do that (or modify the Qemu command line directly) you still need to understand DHCP and what these options do.
